cypress/integration/jsGrowthTableDOM.spec.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+Cypress.Commands.add('clickButton',
+(selector, times) => {
+  for (let n = 0; n < times; n++) {
+    cy.get(selector).click( {force: true} );
+  }
+});
+
+describe('Table', () => {
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
is this needed? 
https://docs.cypress.io/guides/references/best-practices.html#Setting-a-global-baseUrl